src/Stratis.Bitcoin.Features.Wallet/Interfaces/IWalletManager.cs

```diff
@@ -122,22 +122,24 @@ public interface IWalletManager
         /// Gets a collection of addresses containing transactions for this coin.
         /// </summary>
         /// <param name="walletName">The wallet name.</param>
+        /// <param name="accountName">The account name.</param>
         /// <returns>Collection of address history and transaction pairs.</returns>
-        IEnumerable<FlatHistory> GetHistory(string walletName);
+        IEnumerable<AccountHistory> GetHistory(string walletName, string accountName = null);
```

Comment:
Maybe a remark when the accountName is null then show history across all accounts ?

Reply:
done